packages/addons/service/librespot/package.mk

```diff
@@ -19,13 +19,12 @@
 
 PKG_NAME="librespot"
 PKG_VERSION="aa86ebf"
-PKG_SHA256="f2e7d1706afb64759cc23affb8bf6d27b4988ce84b0ed34c156b52eb7e2ca923"
```

Comment:
:)

When bumping package PKG_SHA256 should be commented and when new version is downloaded you see the checksum as part of the download process. Then this checksum is copied back to package.mk.